frontend/app/work_packages/controllers/work-package-details-controller.js

```diff
@@ -172,13 +174,78 @@ module.exports = function($scope,
     return !!($scope.workPackage && $scope.workPackage.embedded.watchers !== undefined);
   };
 
-  function displayedActivities(workPackage) {
+  $scope.isInitialActivity = function(activity, activityNo) {
+    var type = activity.props._type,
+      activities = $scope.activities;
+
+
+    // Type must be Activity
+    if (type !== 'Activity') {
```

Comment:
Oh, alright. Why exactly can a first activity journal be a comment? I assume that happens if you add a note while wp creation?